platform/android/java/lib/build.gradle

```diff
@@ -105,7 +109,7 @@ android {
         def taskName = getSconsTaskName(buildType)
         tasks.create(name: taskName, type: Exec) {
             executable sconsExecutableFile.absolutePath
-            args "--directory=${pathToRootDir}", "platform=android", "target=${releaseTarget}", "android_arch=${defaultAbi}", "-j" + Runtime.runtime.availableProcessors()
+            args "tools=${tools}", "--directory=${pathToRootDir}", "platform=android", "target=${releaseTarget}", "android_arch=${defaultAbi}", "-j" + Runtime.runtime.availableProcessors()
```

Comment:
need a way to programicly set tools by which configuration is ran...

Reply:
The problem is that I want to be able to build the editor with Android Studio, and the play/debug only runs the default build...
But because that it runs the editor gradle and it depends on the native gradle as far as I saw there is no way to send data to a dependency in gradle

Reply:
This should be doable by leveraging [product flavors](https://developer.android.com/studio/build/build-variants#product-flavors)

Reply:
Addressed in https://github.com/thebestnom/godot/pull/2